webpack-stats.json

```diff
@@ -0,0 +1 @@
+{"status":"done","chunks":{"root":[{"name":"root.js","path":"/Users/mbertrand/code/odl-video-service/static/bundles/root.js"},{"name":"root.js.map","path":"/Users/mbertrand/code/odl-video-service/static/bundles/root.js.map"}],"style":[{"name":"style.js","path":"/Users/mbertrand/code/odl-video-service/static/bundles/style.js"},{"name":"style.js.map","path":"/Users/mbertrand/code/odl-video-service/static/bundles/style.js.map"}]}}
```

Comment:
This file shouldn't be checked in, it's generated by webpack to communicate its status to Django via `webpack_loader`